package.json

```
@@ -26,7 +26,7 @@
     "graphql-js"
   ],
   "engines": {
-    "node": "^16.19.0 || ^18.14.0 || >=19.7.0"
+    "node": ">=20.19.0 || >=22.12.0 || >=23.0.0 || >=24.0.0"
```

```suggestion
    "node": "^20.19.0 || ^22.12.0 || >=23.0.0"
```
I believe the >= meaning anything greater than, so 22.11.0 [incorrectly passes for me](https://jubianchi.github.io/semver-check/#/%3E%3D20.19.0%20||%20%3E%3D22.12.0%20||%20%3E%3D23.0.0%20||%20%3E%3D24.0.0/22.11.0) with existing specifiers.

For same reason we only need one >=, as >=23.0.0 implies >=24.0.0